src/layouts/BaseLayout/InstallationHelpBanner/InstallationHelpBanner.spec.jsx

```diff
@@ -0,0 +1,203 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { graphql } from 'msw'
+import { setupServer } from 'msw/node'
+import { MemoryRouter, Route, Switch } from 'react-router-dom'
+
+import { useLocationParams } from 'services/navigation'
+
+import InstallationHelpBanner from './InstallationHelpBanner'
+
+jest.mock('services/navigation')
```

Comment:
Again can you unmock this so we can test the proper use of `useLocationParams` please and thank you.